### ISSUE TITLE
RuntimeError: config must be loaded first

### ISSUE THREAD

**Issue by quanhua92** [NVIDIA/DIGITS#103]
Hi, I use digits with Caffe. I use digits without cudnn. When I add a new dataset, I encounter the below error. What can I do now?

```
127.0.0.1 - - [2015-05-12 17:59:51] "GET / HTTP/1.1" 200 3771 0.042867
127.0.0.1 - - [2015-05-12 17:59:54] "GET /datasets/images/classification/new HTTP/1.1" 200 19170 0.028342
127.0.0.1 - - [2015-05-12 17:59:59] "POST /datasets/images/classification HTTP/1.1" 302 630 0.484615
127.0.0.1 - - [2015-05-12 17:59:59] "GET /datasets/20150512-175959-6472 HTTP/1.1" 200 16048 0.049022
2015-05-12 18:00:00 [20150512-175959-6472] [DEBUG] Parse Folder (train/val) task queued.
2015-05-12 18:00:01 [20150512-175959-6472] [INFO ] Parse Folder (train/val) task started.
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: Traceback (most recent call last):
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: File "/home/robotbase/github/digits/tools/parse_folder.py", line 19, in <module>
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: import digits.log
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: File "/home/robotbase/github/digits/digits/log.py", line 106, in <module>
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: logger = setup_logging()
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: File "/home/robotbase/github/digits/digits/log.py", line 76, in setup_logging
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: if config_option('log_file'):
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: File "/home/robotbase/github/digits/digits/config.py", line 1138, in config_option
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: raise RuntimeError('config must be loaded first')
2015-05-12 18:00:01 [20150512-175959-6472] [WARNING] Parse Folder (train/val) unrecognized output: RuntimeError: config must be loaded first
2015-05-12 18:00:01 [20150512-175959-6472] [ERROR] Parse Folder (train/val) task failed with error code 1
```

**Comment by lukeyeager** [comment]
>File "/home/robotbase/github/digits/digits/config.py", line 1138, in config_option
raise RuntimeError('config must be loaded first')
RuntimeError: config must be loaded first

Hmm, that looks like a bug related to #96, which I closed yesterday.

How did you start DIGITS? With `digits-devserver` or `digits-server`.

**Comment by quanhua92** [comment]
Hi, I tried "digits-devserver", "digits-devserver -c" to change the configuration. But nothing works. However, I have no problem while using the web installer version 1.0.

**Comment by lukeyeager** [comment]
Yeah, I see the problem. I'm not calling `load_config()` at the right time. Thanks for pointing this out, I'll try to get it worked out today.

If you want to use DIGITS today while I'm working on this fix, the `digits-1.1` branch will work for you.
